meshtastic/tests/test_mesh_interface.py

```
@@ -53,7 +57,9 @@ def _import_with_print_color_failure(
         level: int = 0,
     ) -> Any:
         if name == "print_color":
-            raise ImportError("simulated missing print_color") from None  # noqa: TRY003 - intentional test sentinel
+            raise ImportError(
+                "simulated missing print_color"
+            ) from None  # noqa: TRY003 - intentional test sentinel
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)